packages/plugin-core/src/services/EngineAPIServiceV2.ts

```diff
@@ -0,0 +1,216 @@
+import {
+  APIUtils,
+  BulkAddNoteOpts,
+  ConfigWriteOpts,
+  DendronAPI,
+  DEngineClient,
+  DEngineInitResp,
+  DEngineSyncOpts,
+  DHookDict,
+  DLink,
+  DVault,
+  EngineDeleteNotePayload,
+  EngineDeleteOpts,
+  EngineInfoResp,
+  EngineUpdateNodesOptsV2,
+  EngineWriteOptsV2,
+  GetNoteBlocksOpts,
+  GetNoteBlocksPayload,
+  GetNoteOptsV2,
+  GetNotePayload,
+  IntermediateDendronConfig,
+  NoteChangeEntry,
+  NoteProps,
+  NotePropsDict,
+  QueryNotesOpts,
+  RenameNoteOpts,
+  RenameNotePayload,
+  RenderNoteOpts,
+  RenderNotePayload,
+  RespRequired,
+  RespV2,
+  SchemaModuleDict,
+  SchemaModuleProps,
+} from "@dendronhq/common-all";
+import { DendronEngineClient, HistoryService } from "@dendronhq/engine-server";
+import _ from "lodash";
+import { getDWorkspace } from "../workspace";
+
+export class EngineAPIServiceV2 implements DEngineClient {
```

Comment:
was this just for testing? any reason why this couldn't be done in `EngineAPIService`?

Reply:
Yep, just for prototyping; it's been converged again.